pom.xml

```diff
@@ -39,21 +39,21 @@
 
 		<!--Add Queue Item Connector -->
 		<uipath-add-queueItem.def.id>uipath-add-queueItem</uipath-add-queueItem.def.id>
-		<uipath-add-queueItem.def.version>2.0.0</uipath-add-queueItem.def.version>
+		<uipath-add-queueItem.def.version>2.1.2</uipath-add-queueItem.def.version>
```

Comment:
Why don't you just bump the version to 2.1.0 ?

Reply:
will update